t/page/country.rb

```diff
@@ -11,9 +11,9 @@ def country_page(qid)
 end
 
 describe 'Estonia Page' do
-  subject do
-    VCR.use_cassette('Estonia Page') { country_page('Q191') }
-  end
+  around { |test| VCR.use_cassette('Estonia page', &test) }
```

Comment:
There's a slight change in the name of the cassette here - 'Estonia **p**age' - that I think was unintentional.

It's currently having the effect of recording a new cassette, i.e. the interaction with the API happens (you can see the test go slow) and there's a new file created on disk that's not checked into the repository.

The intention here was probably _not_ to change the cassette name so the simple fix is to keep the name as it was.

We could also change the name of the cassette but with these names I was trying to directly match the test description block name so there is some thought behind it 😄

Reply:
Ooops, good catch. Yes, completely unintentional, and as the macOS case insensitive file system doesn't create a new cassette this didn't show up locally for me!

Reply:
>...the macOS case insensitive file system...

😱